x-pack/plugins/security_solution/public/detections/components/rules/pre_packaged_rules/load_empty_prompt.test.tsx

```diff
@@ -161,9 +161,9 @@ describe('LoadPrebuiltRulesAndTemplatesButton', () => {
     await waitFor(() => {
       wrapper.update();
 
-      expect(
-        wrapper.find('[data-test-subj="load-prebuilt-rules"] button').props().disabled
-      ).toEqual(true);
+      expect(wrapper.find('button[data-test-subj="load-prebuilt-rules"]').props().disabled).toEqual(
```

Comment:
Nit: could we just use `[data-test-subj="load-prebuilt-rules"]` w/o `button` as a selector here?